x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlAutoscalingResourceTracker.java

```diff
@@ -259,7 +299,7 @@ static void getMemoryAndProcessors(
                 );
 
                 modelMemoryBytesSum += estimatedMemoryUsage;
-                processorsSum += numberOfAllocations * numberOfThreadsPerAllocation;
+                sumOfCurrentlyExistingProcessors += numTargetAllocations * numberOfThreadsPerAllocation;
```

Comment:
I don't really understand what the point of the processorsSum is. I think it might just be informative? It seems like the elasticsearch-autoscaler doesn't even read this field. Elasticians can check client/ml.go in elasticsearch-autoscaler.